packages/teams-js/test/private/privateAPIs.spec.ts

```diff
@@ -90,15 +90,28 @@ describe('AppSDK-privateAPIs', () => {
     'https://gov.teams.microsoft.us',
     'https://dod.teams.microsoft.us',
     'https://int.teams.microsoft.com',
+    'https://teams.live.com',
     'https://devspaces.skype.com',
+    'https://ssauth.skype.com',
+    'https://local.teams.live.com',
+    'https://local.teams.live.com:8080',
     'https://local.teams.office.com',
-    'https://microsoft.sharepoint.com',
+    'https://local.teams.office.com:8080',
     'https://msft.spoppe.com',
+    'https://microsoft.sharepoint.com',
     'https://microsoft.sharepoint-df.com',
     'https://microsoft.sharepointonline.com',
     'https://outlook.office.com',
     'https://outlook-sdf.office.com',
     'https://retailservices.teams.microsoft.com',
+    'https://www.office.com',
+    'https://word.office.com',
+    'https://excel.office.com',
+    'https://powerpoint.office.com',
+    'https://www.officeppe.com',
+    'https://test.www.office.com',
+    'https://test.www.microsoft365.com',
+    'https://www.microsoft365.com',
     'https://tasks.office.com',
```

Comment:
similar here... not sure if it should be pullled up a bit along with other *.office.com

Reply:
This one I actually want to keep where it is. tasks.office.com isn't actually in the list of validOrigins. It's only allowed here because it's being passed in to the intializeWithContext call as an additional validOrigin by the test (along with https://www.example.com)